packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -39,6 +39,8 @@ const tracestateTagKeyFilter = /[^\x21-\x2b\x2d-\x3c\x3e-\x7e]/g
 // Tag values in tracestate replace ',', '~' and ';' with '_'
 const tracestateTagValueFilter = /[^\x20-\x2b\x2d-\x3a\x3c-\x7d]/g
 const invalidSegment = /^0+$/
+const ddParentIdTagKey = '_dd.parent_id'
+const zeroHex = '0000000000000000'
```

Comment:
The variable name is a bit too literal IMO. For example, if I had a button with a warning color of "FF0000", which is red, the literal name would be `const colorRed` but a more descriptive name would be `const colorWarning`.

What is zeroHex really? Is it the trace ID of a parentless span? If so, a more descriptive name like "rootSpanTraceId" would be better. Otherwise using `zeroHex` is no better than `'0000000000000000'` in the code.